### DISCUSSION TITLE
Cannot add integration

### DISCUSSION THREAD

**Discussion by cb29983** [Tasshack/dreame-vacuum#621] [General]
I cant work this out. I have the Dreamebot Z10 Pro.
I have tried adding the integration (both current version and the BETA) but they both fail. I can add my xiaomi home logon credentials, it loads the next page with name, notifications, map icon set etc, click submit and get "Unknown error occurred" and HA restarts itself.
Any ideas??

**Comment by Tasshack** [comment]
Two pinned issues that you should check first;
https://github.com/Tasshack/dreame-vacuum/issues/122
https://github.com/Tasshack/dreame-vacuum/issues/52

**Reply by cb29983** [reply]
Thanks for the reply @Tasshack 
Apologies, but i still cannot find these pinned "Issues" of my own searching - does not seem to be very user friendly. Have a lovely day.
